swift.html.markdown

```diff
@@ -147,6 +148,11 @@ for value in myArray {
     }
 }
 
+// for loop with indices (array)
```

Comment:
Yes, I'm pretty sure it changed from `indices(array)` to `array.indices()`
I do agree, enumerate is quite handy and should be included as well.